### DISCUSSION TITLE
EXIF metadata removal or addition when converting

### DISCUSSION THREAD

**Discussion by jeffbuzz** [ImageMagick/ImageMagick#5914] [Help] [answered]
### ImageMagick version

7.1.0-55

### Operating system, version and so on

Windows 11 x64

### Description

Are there methods available to remove or only include explicit metadata during export?  Specifically I want to remove all but some specific camera, lens and exposure related EXIF data when exporting images.

Thank you.

**Comment by fmw42** [comment, marked as answer]
-strip and +profile. The first removes most if not all meta data.  The second only removes profiles as specified.

**Reply by fmw42** [reply]
You may want to use EXIFTOOL as it is designed to deal better with meta data.

**Reply by jeffbuzz** [reply]
Thank you very much.
